packages/eslint-config/README.md

```diff
@@ -0,0 +1,45 @@
+# v2 - wip
```

Comment:
Should we update this?

Reply:
We are going to do alpha releases for a while, so still wip